components/graphs/ThresholdGraph.tsx

```diff
@@ -130,14 +130,37 @@ const ThresholdGraph: FC<{ siteUUID: string }> = ({ siteUUID }) => {
     }
 
     const startTime = timeFormatter.format(new Date(graphData[0].datetime_utc));
+    const startDay = dayFormatter.format(new Date(graphData[0].datetime_utc));
+
     const endTime = timeFormatter.format(
       new Date(graphData[numForecastValues - 1].datetime_utc)
     );
+    const endDay = dayFormatter.format(
+      new Date(graphData[numForecastValues - 1].datetime_utc)
+    );
 
     return (
       <div className="flex flex-row justify-between">
-        <p className="text-white text-xs font-medium ml-6">{startTime}</p>
-        <p className="text-white text-xs font-medium mr-6">{endTime}</p>
+        <div>
+          <p className="text-white text-xs sm:text-base font-semibold sm:font-medium ml-3 sm:ml-6">
+            {startTime}
+          </p>
+          <p className="text-white text-xs sm:text-base font-normal ml-3 sm:ml-6">
+            {startDay}
+          </p>
```

Comment:
Can we only show the day if you're looking at tomorrow's forecast?

Reply:
Any reason why not to show the time?

Reply:
Ah my language was confusing, sorry. I mean that we should not show the day if we're looking at today's forecast.

Reply:
Why not include the day? It can strengthen the yellow marker on the graph's indication that we are showing the current day's forecast.

Reply:
I'm for as little text as possible on the dashboard. That way, the text that is there is all easy to parse and notice without having to filter redundant info. I think it is assumed that the graph's start and end are for the current day if the yellow marker is there. We can ask Kalyn about this though.